pkg/handler/repositories.go

```diff
@@ -52,6 +52,7 @@ func RegisterRepositoryRoutes(engine *echo.Group, daoReg *dao.DaoRegistry,
 	addRoute(engine, http.MethodPost, "/repositories/bulk_delete/", rh.bulkDeleteRepositories, rbac.RbacVerbWrite)
 	addRoute(engine, http.MethodPost, "/repositories/", rh.createRepository, rbac.RbacVerbWrite)
 	addRoute(engine, http.MethodPost, "/repositories/bulk_create/", rh.bulkCreateRepositories, rbac.RbacVerbWrite)
+	addRoute(engine, http.MethodPost, "/repositories/:uuid/snapshot/", rh.syncSnapshot, rbac.RbacVerbWrite)
```

Comment:
what do you think about moving this to the snapshot  handler (and maybe renaming it 'create') ?

Reply:
I think that makes sense.
When you say "this" you mean the method, the tests, all of it?  
Or just the route?